### DISCUSSION TITLE
Limit use of elements

### DISCUSSION THREAD

**Discussion by cbaxter7** [vueform/builder#66] [Questions]
I'm looking to limit the usage of specific elements to once per form

In the example below the fields that have already been added to the form would either be removed from the list of elements or removed completely.

Would this be possible?
![image](https://github.com/vueform/builder/assets/130754988/42a6d62d-6b3d-41bf-b89e-a355126789f8)

**Comment by adamberecz** [comment]
It's not a built-in feature, but it can be achieved:
https://stackblitz.com/edit/github-bmamqp?file=src%2FApp.vue

Actually, it's a great idea to add as a core feature, so I've [added it](https://github.com/vueform/builder/discussions/67) to the idea pool.
